examples/reference/indicators/LoadingSpinner.ipynb

```diff
@@ -17,36 +17,45 @@
    "cell_type": "markdown",
    "metadata": {},
```

Comment:
When not setting a `bgcolor` the `label` seems to be out of its place

![image](https://github.com/user-attachments/assets/8b7989fb-6c54-458f-8623-70f9f351bcd4)